.changeset/sixty-spoons-own.md

```diff
@@ -0,0 +1,9 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/core-typings": minor
+"@rocket.chat/model-typings": minor
+"@rocket.chat/models": minor
+"@rocket.chat/rest-typings": minor
+---
+
+Introduced "create contacts" endpoint to the omnichannel
```

Comment:
```suggestion
Introduced "create contacts" endpoint to omnichannel
```

I'd confirm with a PM but I'm pretty sure it's not "the omnichannel" but just "omnichannel"